coverage.rst

```diff
@@ -72,30 +72,54 @@ Using coverage.py
 One of the most popular third-party coverage tools is `coverage.py`_ which
 provides very nice HTML output along with advanced features such as
 :ref:`branch coverage <branch_coverage>`. If you prefer to stay with tools only
-provided by the stdlib then you can by :ref:`using test.regrtest
+provided by the stdlib then you can :ref:`use test.regrtest
 <coverage_by_regrtest>`.
 
-Because the in-development version of Python is bleeding-edge, it is possible
-that the latest release version of coverage.py will not work. In that case you
-should try using the in-development of coverage.py to see if it has been
-updated as needed. To do this you should clone/check out the development version
-of coverage.py::
+
+.. _install_coverage:
+
+Install Coverage
+''''''''''''''''
+
+By default, pip will not install into the in-development version of Python you 
+just built, and this built version of Python will not see packages installed 
+into your default version of Python. One option is to use a virtual environment
+to install coverage::
+
+    ./python.exe -m venv ../cpython-venv
```

Comment:
maybe we could use `$PWD`, example:
```
./python.exe -m venv $PWD-env
source $PWD-env/bin/activate
```

and on Linux, the executable file does not finish with `.exe`, just on Windows and Darwin.